go-controller/pkg/ovn/egressip.go

```diff
@@ -372,7 +371,10 @@ func (e *egressIPMode) getGatewayRouterJoinIP(node string, wantsIPv6 bool) (net.
 	} else {
 		err := utilwait.ExponentialBackoff(retry.DefaultRetry, func() (bool, error) {
 			var err error
-			gatewayIPs, err = util.GetNodeLogicalRouterIPs(node)
+			gatewayIPs, _, err = util.GetNodeGatewayRouterNetInfo(node)
```

Comment:
why do you add a `[]*net.IPNet` return value to the function but then not use it from the only caller of that function?

Reply:
The idea is to use this function in a upcoming PR to unify the retrieval of the join gateway router IP / net. Hence why. But I agree, this patch does not do that yet, so I will remove it.